code/controllers/subsystems/initialization/atlas.dm

```diff
@@ -64,7 +64,8 @@ var/datum/controller/subsystem/atlas/SSatlas
 	setup_multiz()
 
 	QDEL_NULL(maploader)
-
+	if(config.awaymissions && prob(config.awaymissionsprob))
+		find_away_mission()
```

Comment:
If you move this above `QDEL_NULL(maploader)`, you could just use Atlas' maploader instance instead of creating a new one in `find_away_mission()`.